lib/Twig/Cache/Null.php

```diff
@@ -45,6 +47,9 @@ public function write($key, $content)
      */
     public function load($key)
     {
+        eval('?>'.$this->buffer[$key]);
```

Comment:
Does twig provide any protection against loading it twice already? In case yes, this looks basically how I expect it to look like. Thank you a lot for the very quick response!